### ISSUE TITLE
[REQUEST] Add ThisVid.com

### ISSUE THREAD

**Issue by NikitaForsov** [AAndyProgram/SCrawler#112]
Please add ThisVid.com. This site is in the top 10 ranking of porn sites.

**Comment by username112299** [comment]
Поддерживаю

**Comment by NikitaForsov** [comment]
Ого, тут можно писать по-русски?
Тогда пожалуйста добавьте сайт ThisVid.com.

**Comment by username112299** [comment]
Конечно. Этот гомосек пользуется гугл переводчиком.

**Comment by NikitaForsov** [comment]
Зачем ты так пишешь?

**Comment by username112299** [comment]
Как так?

**Comment by username112299** [comment]
Зачем ты его гомосеком называешь?

**Comment by NikitaForsov** [comment]
Зачем ты его гомосеком называешь?

**Comment by username112299** [comment]
А ты не знал что он пидор? Он сам в своей программе написал что он пидорас. И иконка у программы пидорская.

**Comment by NikitaForsov** [comment]
Нет, конечно я знаю что он пидор, не думал что его можно пидорасом называть. Но то что он пидорас меня не напрягает. Мне нравится программа. Хотя о том что он пидор не обязательно писать в программе и иконку можно было бы другую подобрать. А его не напрягает что его пидором зовут?

**Comment by username112299** [comment]
А как его еще называть если он пидор?

**Comment by NikitaForsov** [comment]
Ну так то да, пидор есть пидор.

**Comment by username112299** [comment]
>А его не напрягает что его пидором зовут?

Я общался с ним в дискорде там все время его пидором называл, ему норм, он опущеный. Я его даже спрашивал норм ему что я его зову пидором он ответил что норм.

**Comment by NikitaForsov** [comment]
Даже так? Ну тогда ладно.

**Comment by username112299** [comment]
Замени в программе информацию на "Created by fag person Andy (fagot)"

**Comment by AAndyProgram** [comment]
@NikitaForsov, @username112299
You are banned!
